tests/environments/test_environments_builders.py

```diff
@@ -157,10 +157,17 @@ def test_environment_feature_states_are_filtered_correctly():
     # When
     build_environment_model(mock_environment)
 
-    # Then
-    mock_feature_states.filter.assert_called_once_with(
+    fs_call_without_segment_or_identity = mock.call(
```

Comment:
I think we should add some comments to explain what's happening / why we're expecting this?

Reply:
we are getting rid of it here #20